server/utils/serviceApp.js

```diff
@@ -9,6 +9,6 @@ var router = express.Router();
 
 router.use(express.static('./client'));
 router.use(express.static('./.tmp'));
-router.use(express.static('./bower_components'));
+router.use(express.static('./node_modules'));
```

Comment:
Uh, I guess that would be me :D 

Ok, I'm seeing they're now copied to `public/ui/service/vendor/{no-vnc,spice-html5-bower}`.. The tricky part here is that we need to find them on the same url both in devel mode (served by node) and in production mode (served by apache) .. but that can be acomplished by redirecting `/ui/service/vendor` to `/node_modules` in node, and assuming the `/ui/service/vendor/...` URL, I think. 

And then, the only change needed outside manageiq-ui-service would be https://github.com/ManageIQ/manageiq/blob/3921e87915b5a69937b9d4a70bb24ab71b97c165/gems/pending/util/miq_apache/miq_apache.rb#L209 .. that `bower_components` was there for exactly that reason (and only that reason, so you should be able to safely change that to `vendor`)